jvb/src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -235,13 +234,19 @@ public Conference(Videobridge videobridge,
                 try
                 {
                     // TODO: we can avoid reaching into Videobridge here by merging VideobridgeShim into ConferenceShim
+                    long start = System.currentTimeMillis();
                     IQ response = videobridge.handleColibriConferenceIQ(this, request.getRequest());
-                    long delay = System.currentTimeMillis() - request.getReceiveTime();
-                    if (delay > 100)
+                    long end = System.currentTimeMillis() - request.getReceiveTime();
+                    long processingDelay = end - start;
+                    request.getProcessingDelayStats().addDelay(processingDelay);
+
+                    long totalDelay = end - request.getReceiveTime();
+                    if (totalDelay > 100)
                     {
-                        logger.warn("Took " + delay + " ms to handle IQ: " + request.getRequest().toXML());
+                        logger.warn("Took " + totalDelay + " ms to handle IQ (processing took "
```

Comment:
I'd argue that the warning should be based on processing delay, not total delay?